### DISCUSSION TITLE
Is there any way to customize login page?

### DISCUSSION THREAD

**Discussion by camenduru** [jupyter/notebook#6676] [Q&A]
I am sorry I think I asked wrong place I should ask here https://github.com/jupyter-server/jupyter_server

**Comment by jtpio** [comment]
> I am sorry I think I asked wrong place I should ask here https://github.com/jupyter-server/jupyter_server

Right this is more likely related to Jupyter Server if you are using the Notebook 7 pre-releases.

This issue looks related: https://github.com/jupyter-server/jupyter_server/issues/933

**Reply by camenduru** [reply]
thanks ❤
